app/controllers/UploadController.scala

```diff
@@ -65,7 +65,7 @@ class UploadController @Inject()(val controllerComponents: SecurityComponents,
 
       uploadStatus match {
         case Some("InProgress") =>
-          Ok(views.html.uploadInProgress(consignmentId, pageHeading, request.token.name)).uncache()
+          Ok(views.html.uploadInProgress(consignmentId, pageHeading, request.token.name, request.token.isJudgmentUser)).uncache()
```

Comment:
Same here but in reverse, isJudgmentUser is always true.

Reply:
good point